src/framework/components/element/component.js

```diff
@@ -1563,10 +1592,18 @@ class ElementComponent extends Component {
         this.off();
     }
 
-    // recalculates
-    // localAnchor, width, height, (local position is updated if anchors are split)
-    // assumes these properties are up to date
-    // _margin
+    /**
+     * recalculates
```

Comment:
If we make this jsdocs, it'd be good to format it nicer as well (even when private)

Reply:
Okay, I worked it out a little bit: 
![image](https://user-images.githubusercontent.com/5236548/192483591-3e9deb8d-dc96-4dbf-85f2-07cf1463d1f9.png)

I renamed `localAnchor` to `_localAnchor` aswell, becaues there is no getter for `localAnchor`.